examples/common/text_data.py

```diff
@@ -84,81 +88,63 @@ def __init__(self,
                          batch_size=batch_size)
         self.tokenizer_name = tokenizer_name
         self.max_seq_len = max_seq_len
-        self.group_method = group_method
 
         # Build tokenizer
         os.environ['TRANSFORMERS_NO_ADVISORY_WARNINGS'] = '1'
+        os.environ['TOKENIZERS_PARALLELISM'] = 'false'
         self.tokenizer = transformers.AutoTokenizer.from_pretrained(
             self.tokenizer_name)
```

Comment:
```suggestion
        self.tokenizer = transformers.AutoTokenizer.from_pretrained(
            tokenizer_name)
```
Not sure why we set `self.tokenizer_name = tokenizer_name` in line 89.